gio/src/auto/initable.rs

```diff
@@ -0,0 +1,67 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::Cancellable;
+use glib::object::IsA;
+use glib::translate::*;
+use std::fmt;
+use std::ptr;
+
+glib::wrapper! {
+    #[doc(alias = "GInitable")]
+    pub struct Initable(Interface<ffi::GInitable, ffi::GInitableIface>);
+
+    match fn {
+        type_ => || ffi::g_initable_get_type(),
+    }
+}
+
+impl Initable {
+    //#[doc(alias = "g_initable_new")]
+    //pub fn new<P: IsA<Cancellable>>(object_type: glib::types::Type, cancellable: Option<&P>, error: &mut glib::Error, first_property_name: Option<&str>, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> glib::Object {
+    //    unsafe { TODO: call ffi:g_initable_new() }
+    //}
+
+    //#[doc(alias = "g_initable_new_valist")]
+    //pub fn new_valist<P: IsA<Cancellable>>(object_type: glib::types::Type, first_property_name: &str, var_args: /*Unknown conversion*//*Unimplemented*/Unsupported, cancellable: Option<&P>) -> Result<glib::Object, glib::Error> {
+    //    unsafe { TODO: call ffi:g_initable_new_valist() }
+    //}
```

Comment:
You can disable both new_valist/new as they are not bindings friendly. Probably rename newv to new as well

Reply:
Done in new version